File: src/main/kotlin/inflearn/java/ch01/_1_2_대소문자_변환.java
package inflearn.java.ch01;

import java.util.Scanner;

public class _1_2_대소문자_변환 {

    public static String solution(String str) {
        StringBuilder answer = new StringBuilder();

        for (char ch : str.toCharArray()) {
            if(Character.isLowerCase(ch))
                answer.append(Character.toUpperCase(ch));
            else
                answer.append(Character.toLowerCase(ch));
        }

        return answer.toString();
    }

    public static void main(String[] args) {
        Scanner in = new Scanner(System.in);
        String input = in.next();

        System.out.println(solution(input));
    }
}

File: src/main/kotlin/inflearn/kotlin/ch01/1_2 대소문자 변환.kt
package inflearn.kotlin.ch01

import java.util.*

object `1_2 대소문자 변환` {
    fun solution(str: String): String {
        val answer = StringBuilder()
        for (ch in str.toCharArray()) {
            if (ch.isLowerCase())
                answer.append(ch.uppercaseChar())
            else
                answer.append(ch.lowercaseChar())
        }
        return answer.toString()
    }

    @JvmStatic
    fun main(args: Array<String>) {
        val `in` = Scanner(System.`in`)
        val input = `in`.next()
        println(solution(input))
    }
}